packages/core/src/interpreter.ts

```diff
@@ -345,8 +345,12 @@ export class Interpreter<
         listener(errorEvent.data.data);
       });
     } else {
-      this.stop();
-      throw errorEvent.data.data;
+      if (this.parent) {
```

Comment:
I think it's important that this branch still stops the current machine - if I understand correctly this whole function can only be called from within `service.send` and is called when there are no explicit error transitions defined in the machine. In such a case I would expect the machine with the unhandled error to be stopped by the XState's runtime

Reply:
More than that - I think it makes sense to perform this logic regardless of the `this.errorListeners`, even if there is an error listener we should still stop the current machine and forward the event to the parent (if there is any)

Reply:
it sounds good to me, let me refactor it